README.md

```diff
@@ -219,9 +221,174 @@ console.log(records)
 
 In this example, the `Client.query` method returns an array that contains each record that matches the query.
 
+## Read a record by record id 
+
+```
+const read = async (client) => {
+
+    // The Client.query method is used in this example to get a record id.
```

Comment:
I know the example is intended to be runnable, but I would suggest we make the example more abstract and make the record ID a prerequisite similar to how a Client is a prerequisite rather than user query.

Reply:
We can do this when I get the examples running.  I don't think that's going to fit into this sprint.